easybuild/easyblocks/i/imkl.py

```diff
@@ -227,16 +227,29 @@ def post_install_step(self):
             # determine whether we're using a non-Intel GCC-based or PGI-based toolchain
             # can't use toolchain.comp_family, because of dummy toolchain used when installing imkl
             if get_software_root('icc') is None:
-                if get_software_root('PGI'):
-                    compopt = 'compiler=pgi'
-                elif get_software_root('GCC'):
+                if get_software_root('GCC'):
```

Comment:
I was deliberately careful about this because I was afraid the underlying GCC would be picked up